### ISSUE TITLE
Errors when trying to run on a RPI3B+

### ISSUE THREAD

**Issue by linusnyren** [tillsteinbach/VWsFriend#93]
HI!
This looks really great!

I'm trying to get this to work on my RPI to integrate into homekit.
But after running docker-compose -f docker-compose-homekit-host.yml --env-file ./myconfig.env up
I get the following error.
`vwsfriend_1   | 2021-10-27T19:51:08+0000:INFO:vwsfriend_base:vwsfriend 0.9.3 (using WeConnect-python 0.21.5)
vwsfriend_1   | 2021-10-27T19:51:08+0000:INFO:weconnect:Could not use token from file /tmp/weconnect.token ([Errno 2] No such file or directory: '/tmp/weconnect.token')
vwsfriend_1   | The futex facility returned an unexpected error code.
vwsfriend_1   | Aborted (core dumped)
vwsfriend_vwsfriend_1 exited with code 134`

Is this something simple i'm missing?
It works great locally on my Mac with docker-compose --env-file ./myconfig.env up

**Comment by tillsteinbach** [comment]
I changed from Ubuntu 21.04 to 21.10. to try if it works petter for the raspberry. Could someone of the Raspberry users try if the edge version works? Just add this to your env file and restart with docker-compose:
`VWSFRIEND_VERSION=edge`

**Comment by linusnyren** [comment]
Hi!
Of course I'll give it a try, will get back shortly!

**Comment by linusnyren** [comment]
Hi again!
It seems to work great, it builds and works.
My car seem to be offline now unfortunately so i don't know how it would work when receiving data.
But i suppose it would work!
It refreshes the token succesfully!
<img width="1226" alt="image" src="https://user-images.githubusercontent.com/36070811/139728331-8267376b-d6d3-421e-beaf-de9bb4f7e085.png">

**Comment by tillsteinbach** [comment]
That's good: just to be sure, does it say: **develop** on the bottom of the webpage? I want to be sure that you really have the latest testing version.

**Comment by linusnyren** [comment]
Hi!
Yes it says develop!

Another question, i tried looking at the weconnect-pythonapi project to find what kind of values i can extract.
Is it possible to get lock status? I have a bad habit of forgetting to lock the car, don't have the keyless version :(

**Comment by tillsteinbach** [comment]
Yes, but not for the ID cars yet. For my Passat GTE I get this:
```
[accessStatus] (last captured 2021-11-01T10:43:39+00:00)
		Overall Status: safe
		Doors: 6 items
			bonnet: closed
			frontLeft: closed
			frontRight: closed
			rearLeft: closed
			rearRight: closed
			trunk: closed
		Windows: 6 items
			frontLeft: closed
			frontRight: closed
			rearLeft: closed
			rearRight: closed
			roofCover: unsupported
			sunRoof: closed
```
So for the ID we have to wait a little longer. But I'm very sure this will come, because the WeConnect ID App can already show that the car is locked

**Comment by linusnyren** [comment]
Oh!
That looks perfect!
Seems like i need to keep an eye on this project.
Unfortunately I'm not familiar with python.

I have a loose plan to port this functionality to nodejs to use with the homebridge framework, but i don't know if i'll ever get the time.

**Comment by tillsteinbach** [comment]
There is good new for you: https://github.com/nightsha-de/npm-vwconnectapi

**Comment by tillsteinbach** [comment]
The changes are now included in 0.9.5 you can remove the VWSFRIEND_VERSION setting again.
